.github/2.0/cookbooks/Document.md

```diff
@@ -800,6 +823,23 @@ doc.tags__dimensions__weight
 10.0
 ```
 
+This can also be used to access nested attributes of repeated `Document` fields such as `scores` or `evaluations`.
+
+```python
+from jina import Document
+
+doc = Document()
+doc.scores.add()
+doc.scores[0].value = 50
+doc.scores.add()
```

Comment:
this `doc.scores.add()` looks very `protobuf`, the syntax is a bit strange in python

Reply:
Yes, I agree, but to explain the access of repeated fields is the best way I found